godtossing/Pos/Ssc/GodTossing/Arbitrary.hs

```diff
@@ -136,9 +136,6 @@ instance Arbitrary Opening where
 
 instance Arbitrary VssCertificate where
     arbitrary = mkVssCertificate <$> arbitrary <*> arbitrary <*> arbitrary
-    shrink VssCertificate {..} = [ VssCertificate { vcExpiryEpoch = shrunkEpoch, .. }
```

Comment:
Write a comment instead, so that no one thinks that you just forgot to write `shrink`.